### PR TITLE
Check for best coding practices

### PR DESCRIPTION
**Pull request by aristides1000**
Please coding partner check if I have complied with the best coding practices:

- [x] [https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md](url).
- [x] [https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md](url).
- [x] [https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md](url).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  return JSON.parse(localStorage.getItem('activities')) || activities;
+}
+
+function displayTasks() {
+  ul.innerHTML = '';
+  const activities = loadDataLocalStorage();
+
+  for (let i = 0; i < activities.length; i++) {
+    const li = document.createElement('li');
+    const checkIn = checked(activities, i);
```
Comment (rotimiazeez) [inline src/index.js:29]: I suggest you use a higher-order method e.g (forEach) instead of the for loop as it makes your javaScript "KISS"

Comment (rotimiazeez) [inline src/index.js]: I suggest you use a higher-order method e.g (forEach) instead of the for loop as it makes your javaScript "KISS"

**Review by rotimiazeez — COMMENTED** [review]
## Hi @aristides1000 🙋‍♂️
You are blazing through
Alt Text

### Worth Commending
- No linter error ✔️ 
- HTML and CSS best practices ✔️ 
- JavaScript best practices ✔️ 
- I can add, remove and update the list. 👍🏽 


### Suggested Change(s) 🔗
- Kindly check the inline comment for suggested change(s)


### Optional suggestion 

> Kindly make the README.md more descriptive to summarize the project.
You have a wonderful and well-structured code 👌. Kindly make the suggested changes and you are good to go 🚀

File: src/index.js (hunk 1 above)
Comment (aristides1000) [reply src/index.js]: Thanks for that restart Azeez, I'm going to work on it

**Comment by aristides1000** [comment]
Change for loop with foreach loop

**Comment by rotimiazeez** [comment]
## Hi @aristides1000 🙋‍♂️
### Congratulation
Alt Text

### Worth Commending
- No linter error ✔️ 
- HTML and CSS best practices ✔️ 
- JavaScript best practices ✔️ 
- I can add, remove and update the list. 👍🏽 


Your project is now completed.🚀

**Pull request merged**